ports/ironferret/port.json

```diff
@@ -0,0 +1,42 @@
+{
+    "version": 3,
+    "name": "ironferret.zip",
+    "items": [
+        "IRON FERRET.sh",
+        "ironferret/"
+    ],
+    "items_opt": null,
+    "attr": {
+        "title": "IRON FERRET",
+        "porter": [
+            "MadShmupper"
+        ],
+        "desc": "IRON FERRET is a small game where you play as a ferret riding a tank. Ultimately it's one giant escort mission, but at least your escortee is much more tankier than you and can shoot bullets.",
+        "desc_md": null,
+        "inst": "Ready To Run",
+        "inst_md": null,
+        "genres": [
+            "action",
+            "puzzle",
+            "other"
+        ],
+        "image": null,
+        "rtr": true,
+        "exp": false,
+        "runtime": null,
+        "reqs": [],
+        "arch": [
+            "armhf"
+        ]
+    },
+    "status": {
```

Comment:
Need to get rid of the "status" and "files" keys. :)

Reply:
Ok